components/Navbar.vue

```diff
@@ -257,7 +258,15 @@ export default class NavbarMenu extends mixins(
   }
 
   get showSearchOnNavbar(): boolean {
-    return !this.isLandingPage || !this.showTopNavbar
+    return !this.isLandingPage || !this.showTopNavbar || this.isBurgerMenuOpened
+  }
+
+  get disableScroll() {
```

Comment:
Setting a get function to a click handler is not proper.
You could use `Watch` method to listen the change of the `isBurgerMenuOpened`.

Reply:
I have updated `disableScroll` function as Watch method.